atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -459,10 +462,11 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
                                 internalKeyValueService,
                                 keyValueServiceManager,
                                 config().transactionKeyValueService().get(),
-                                runtimeConfig().get().map(optionalConfig -> optionalConfig
+                                runtimeConfig.map(optionalConfig -> optionalConfig
                                         .get()
                                         .transactionKeyValueService()
-                                        .get()),
+                                        .orElseThrow(() -> new SafeIllegalArgumentException(
+                                                "TransactionKeyValueServiceRuntimeConfig must be provided"))),
```

Comment:
Well spotted - this does look like a mistake we made in #6984 😅 

That said, I'm not sure this is the right way to fix it - we already wrap the runtime config in a Refreshable on lines 372-373. Notice that other code around this uses `runtime` rather than `runtimeConfig()` or `runtimeConfigSupplier()`. So I think here this should just be `runtime.map(config -> config.tkvs().orElseThrow(...))`.